crypto/multisig.go

```diff
@@ -234,9 +234,9 @@ func MultisigVerify(msg Hashable, addr Digest, sig MultisigSig) (verified bool,
 	return true, nil
 }
 
-// MultisigBatchVerify verifies an assembled MultisigSig.
+// MultisigBatchVerifyPrep verifies an assembled MultisigSig.
 // it is the caller responsibility to call batchVerifier.verify()
-func MultisigBatchVerify(msg Hashable, addr Digest, sig MultisigSig, batchVerifier *BatchVerifier) (verified bool, err error) {
+func MultisigBatchVerifyPrep(msg Hashable, addr Digest, sig MultisigSig, batchVerifier *BatchVerifier) (verified bool, err error) {
```

Comment:
if we are trying to better describe the functionality I think we should avoid using the word `Verify` here since we are aggregating the signature and after this function ends, signatures are not yet verified.

Reply:
Well, It is meant to convey "Preparation" for "Batch verify". The "Batch verify" is the product being prepared for.
I am open to suggestions, but this needed to be done to match the other renaming instances.

Reply:
Maybe just remove the word `verify` just to avoid confustion